src/Components/HomePage.tsx

```diff
@@ -24,17 +26,18 @@ const HomePage = () => {
     }
   return (
     <>
+      <ScrollToTop />
       <Navbar />
     <div className='container mt-3'>
         <div className='row'>
         {products.map((product: any) => {
             return <>
-            <div key={product.id} className='col-md-3 my-3 text-center'>
-                <div className="card" style={{width: "20rem"}} >
+            <div key={product.id} className='col-md-3 my-3 text-center' style={{width: "20rem", maxHeight: "500px"}}>
```

Comment:
is using inline styling good?

Reply:
No sir, but in some cases I don't find bootstrap classes for styling.